src/core/Trigger.helper.js

```
@@ -0,0 +1,46 @@
+export default class TriggerHelper {
+  static async timeout (ms) {
+    return new Promise(r => setTimeout(r, ms));
+  }
+
+  static async action (producer) {
+    try {
+      await producer.collect();
+    } catch (cause) {
+      console.log(cause);
```

DoD - do we want to fail silently?